src/applications/claims-status/tests/components/DocumentRequestPage.unit.spec.jsx

```diff
@@ -29,296 +29,391 @@ const defaultProps = {
-      <DocumentRequestPage {...defaultProps} trackedItem={trackedItem} />,
-    );
+    it('should render due date info', () => {
+      const trackedItem = {
+        status: 'NEEDED_FROM_YOU',
+        suspenseDate: '2010-05-10',
+      };
 
-    expect(tree.subTree('.optional-upload')).not.to.be.false;
-  });
+      const { context } = renderWithRouter(
+        <Provider store={getStore(false)}>
+          <DocumentRequestPage {...defaultProps} trackedItem={trackedItem} />,
+        </Provider>,
+      );
 
-  it('should handle submit files', () => {
-    const trackedItem = {
-      status: 'NEEDED_FROM_YOU',
-      suspenseDate: '2010-05-10',
-    };
-    const onSubmit = sinon.spy();
-    const tree = SkinDeep.shallowRender(
-      <DocumentRequestPage
-        {...defaultProps}
-        trackedItem={trackedItem}
-        submitFiles={onSubmit}
-      />,
-    );
-    tree.subTree('AddFilesFormOld').props.onSubmit();
-    expect(onSubmit.called).to.be.true;
-  });
+      expect($('.due-date-header', context)).to.exist;
+    });
 
-  it('should handle submit files lighthouse', () => {
-    const submitFilesLighthouse = sinon.spy();
-
-    const trackedItem = {
-      status: 'NEEDED_FROM_YOU',
-      suspenseDate: '2010-05-10',
-    };
-    const { container, rerender } = renderWithRouter(
-      <DocumentRequestPage
-        {...defaultProps}
-        trackedItem={trackedItem}
-        submitFilesLighthouse={submitFilesLighthouse}
-        documentsUseLighthouse
-      />,
-    );
-
-    // Check the checkbox
-    $('va-checkbox', container).__events.vaChange({
-      detail: { checked: true },
+    it('should render optional upload alert', () => {
+      const trackedItem = {
+        status: 'NEEDED_FROM_OTHERS',
+        suspenseDate: '2010-05-10',
+      };
+      const { context } = renderWithRouter(
+        <Provider store={getStore(false)}>
+          <DocumentRequestPage {...defaultProps} trackedItem={trackedItem} />,
+        </Provider>,
+      );
+
+      expect($('.optional-upload', context)).to.exist;
     });
 
-    // Create a file
-    const file = {
-      file: new File(['hello'], 'hello.jpg', {
-        name: 'hello.jpg',
-        type: fileTypeSignatures.jpg.mime,
-        size: 9999,
-      }),
-      docType: { value: 'L029', dirty: true },
-      password: { value: '', dirty: false },
-      isEncrypted: false,
-    };
-
-    rerenderWithRouter(
-      rerender,
-      <DocumentRequestPage
-        params={params}
-        claim={claim}
-        trackedItem={trackedItem}
-        submitFilesLighthouse={submitFilesLighthouse}
-        uploadField={{ value: null, dirty: false }}
-        resetUploads={() => {}}
-        files={[file]}
-        documentsUseLighthouse
-        clearNotification={() => {}}
-      />,
-    );
-
-    fireEvent.click($('.submit-files-button', container));
-    expect(submitFilesLighthouse.called).to.be.true;
-  });
+    // it('should handle submit files', () => {
```

Comment:
@jerekshoe had trouble getting this test to refactor. let me know your thoughts and if I should even keep this test!